crates/bevy_core_pipeline/src/main_pass_3d.rs

```diff
@@ -134,12 +134,14 @@ impl Node for MainPass3dNode {
                 })],
                 depth_stencil_attachment: Some(RenderPassDepthStencilAttachment {
                     view: &depth.view,
-                    // NOTE: For the transparent pass we load the depth buffer but do not write to it.
+                    // NOTE: For the transparent pass we load the depth buffer. There should be no
+                    // need to write to it, but store is set to `true` as a workaround for issue #3776,
```

Comment:
Please include a URL link here: it's much more convenient.

Reply:
Done.